services/market/app.js

```diff
@@ -50,7 +50,8 @@ app.addJobs(path.join(__dirname, 'jobs'));
 
 // Run the application
 app.run().then(() => {
-	logger.info(`Service started ${packageJson.name}`);
+	const targetPairs = config.market.targetPairs || 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF';
```

Comment:
The list of tokens is mentioned twice. Please make it based on the definition in the config.